test/test_controller.py

```diff
@@ -285,7 +285,7 @@ def test_backup_state_from_removed_site_is_removed(default_backup_site, mysql_em
     for file_name in fake_file_names:
         assert not os.path.exists(file_name)
 
-
+@pytest.mark.skip(reason="This seems very flaky. See BF-1198")
```

Comment:
I don't think we should have references to BF-* in this repo.

Reply:
uh, OK, that I didn't know, will double-check. Is this only for source code? Or for PRs/commit messages as well?